### PR TITLE
[brightid] Revising the BrightID Strategy

### PR DESCRIPTION
**Pull request by misterplus**
For context:
image
As approved by @bonustrack 
image

Changes proposed in this pull request:
- Revise the current "brightid" strategy to use Official Registries created by BrightID for snapshot while preserving backward compatibility
- Change the "brightid" strategy description to better guide DAOs that need a private registry
- Add a new example for the new use case

I'm not very familiar with how snapshot works under the hood, so my implementation might not be best practices, feel free to change the details as long as it matches the following workflow:
flow (2)

This PR will stay a draft until we finished the work on a frontend site for the Official Registry (so regular users can use it properly), and add its link to the strategy description.

### CODE REVIEW

**Comment by misterplus** [comment]
The frontend site is finished, currently under internal testing, you can view the source code here: https://github.com/SongADAO/songaday-idchain-registration

**Comment by misterplus** [comment]
SongADAO had finished a frontend site for registering with the official BrightID contract: https://idchain.songadao.org/idchain-registration/
Souce code is available here: https://github.com/SongADAO/songaday-idchain-registration
Technically anyone can implement a similar site for their DAO, or just snapshot users in general.

File: src/strategies/brightid/index.ts [hunk 1]
```diff
@@ -16,12 +20,13 @@ export async function strategy(
   snapshot
 ) {
   const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const registry = options.registry[0] == "v" ? official[options.registry] : options.registry;
   const response = await multicall(
-    network,
+    options.registry[0] == "v" ? "74" : network,
```
Comment (ChaituVR) [inline src/strategies/brightid/index.ts:25]: Any idea why this fail? 
```
    TypeError: Cannot read properties of undefined (reading 'toLowerCase')

      14 |     const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
      15 |     const registry = options.registry[0] == "v" ? official[options.registry] : options.registry;
    > 16 |     const response = await (0, utils_1.multicall)(options.registry[0] == "v" ? "74" : network, provider, abi, addresses.map((address) => [
         |                            ^
      17 |         registry,
      18 |         'isVerifiedUser',
      19 |         [address]
```

Try running `yarn test --strategy=brightid --more=500` in your local and make sure it pass all tests

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by misterplus** [comment]
@ChaituVR I've updated the branch with a version that passes `yarn test --strategy=brightid --more=500`, please take a look.

**Review by bonustrack — APPROVED** [review]
Looks good!

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**